test/gtest/ucp/test_ucp_request.cc

```diff
@@ -459,10 +429,20 @@ class test_proto_reset : public ucp_test {
         }
     }
 
-    bool no_rma_transport()
+    unsigned count_rma_transports(uint64_t capability)
```

Comment:
maybe rename to count_tl_with_caps since the function itself is not really specific to RMA caps